ServerCore/Helpers/SubmissionEvaluator.cs

```diff
@@ -0,0 +1,356 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+
+namespace ServerCore.Helpers
+{
+

```

Comment:
Lots of bonus blank lines in this file. A quick formatting pass would be nice, but isn't critical